webapi/src/main/twirl/queries/sparql/v1/createValue.scala.txt

```diff
@@ -83,6 +83,9 @@ DELETE {
 
                     *@
                     ?resource <@{linkUpdate.linkPropertyIri}Value> ?linkValue@linkValueIndex .
+
+                    @* Delete the UUID from the current version of the link value, because the new version will store it. *@
```

Comment:
if this creates a new value, why would there already be a UUID?

Reply:
If we're creating a new text value, we may have to update existing link values for standoff links. If there are existing link values, they will have UUIDs.

Reply:
Ah, I remember! There is only one link but the ref count could be more than one

Reply:
Exactly. So much fun. :)

Reply:
posttraumatic standoff disorder -> PTSD

Reply:
😜

Reply:
It was not meant as a joke. I am actually suffering from it ;-)